implementations/elixir/ockam/ockam/test/ockam/secure_channel_test.exs

```diff
@@ -265,18 +245,255 @@ defmodule Ockam.SecureChannel.Tests do
                    10_000
   end
 
+  test "local secure channel" do
+    {:ok, vault} = SoftwareVault.init()
+    {:ok, alice, alice_id} = Identity.create(@identity_impl)
```

Comment:
Is it worthwhile to hoist some of these up to setup functions if we're always scrutinizing a pair of Alice/Bob identities in each test body?

Reply:
done.   There is still two "styles" of tests on this file,   one that was here originally and the other for tests moved from the identity secure channel suite.
That can be addressed separately